components/raftstore/src/store/peer.rs

```diff
@@ -982,6 +983,10 @@ where
         // becoming a leader.
         self.maybe_update_read_progress(reader, progress);
 
+        // Update leader info
+        self.read_progress
```

Comment:
I find it's confusing. There are `maybe_update_read_progress` and `read_progress`, and they refer to different things.

Reply:
How about rename the `maybe_update_read_progress` to `maybe_update_leader_read_progress` or `maybe_update_lease_read_progress`?

Reply:
How about renaming `RegionReadProgress` to `SafeReadTracker`?

Reply:
Renaming `RegionReadProgress` would be a huge change, if just to resolve the naming conflict I think it is better to change the less used one.

Reply:
Leave a TODO for now, will change it later.